### ISSUE TITLE
Arrangements for data paths and log filtering for Instrument machines

### ISSUE THREAD

**Issue by ChrisM-S** [ISISComputingGroup/IBEX#4258]
As a data administrator and systems administrator, I would like to make sure that all data produced on an instrument computer  follows a rapid preservation pathway off the instrument (if the data is "important"), and if necessary some data may be retained in a fixed size cache (or database) on the instrument for control purposes.

This ticket is related to #3709 part II in that a side effect of properly managing the data paths from/to the instrument is that data does not pool on the instruments. Also as a result of this the instrument resources become well managed and the resource management process automated.  A further and essential result of this is that the real-time resilience of the instrument control system is maintained (i.e. suitable attention to unplanned "burst mode" data issues which may rapidly eat disk space and for which a large unused disk space buffer is required).

**Comment by Tom-Willemsen** [comment]
Presumably this ticket is a meeting to discuss:

- What kinds of data currently accumulate on instruments
  * log files
  * inst archive
  * ioc log database
  * any others?
- For each of the items above, what approaches we can use to move the data away from the instrument control machines automatically.

Based on the outcome of the meeting we would then create separate ticket(s) to cover any required implementation.

**Comment by Kevin-Woods-Tessella** [comment]
Hold on a minute!  We already have a process for moving data off an instrument computer (the process may or may not be documented, but it exists nonetheless).  Before we start discussing anything, we need a shared understanding of the current process.  I would like to see the current process documented before we do anything new.  If the documentation already exists, just add a link to the document to this ticket.  If the documentation does not exist, it needs to be created (it can be a Word document, or a wiki document (preferred)).
Once we have written documentation, we can review it to determine:
1. how much of the current process is still valid
1. what needs to change

**Comment by John-Holt-Tessella** [comment]
See https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Data-Generation-and-Storage for documentation describing above.

**Comment by kjwoodsISIS** [comment]
The [above page](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Data-Generation-and-Storage) defines a number of data types and actions to take for each.  How many of these actions are now implemented?  We should create individual tickets to implement each outstanding action.  Each ticket can then be implemented as and when we have the time to do so.

**Comment by kjwoodsISIS** [comment]
Moving this ticket out of SPRINT_2019_04_25 into "awaiting".

**Comment by GRyall** [comment]
I believe we addressed this in #8321
